src/cryptography/hazmat/backends/openssl/ec.py

```diff
@@ -200,6 +200,23 @@ def private_numbers(self):
             public_numbers=self.public_key().public_numbers()
         )
 
+    def private_bytes(self, encoding, format, encryption_algorithm):
+        evp_pkey = self._backend._lib.EVP_PKEY_new()
+        assert evp_pkey != self._backend._ffi.NULL
+        evp_pkey = self._backend._ffi.gc(
+            evp_pkey, self._backend._lib.EVP_PKEY_free
+        )
+        res = self._backend._lib.EVP_PKEY_set1_EC_KEY(evp_pkey, self._ec_key)
```

Comment:
This is a question dreid asked awhile back in an issue I can't find at the moment. The short answer is "we didn't need it until now". We can hoist it up into the init now though if we want.